python/nano/tutorial/notebook/inference/tensorflow/accelerate_tensorflow_inference_onnx.ipynb

```diff
@@ -1,89 +1,749 @@
 {
   "cells": [
     {
+      "attachments": {},
       "cell_type": "markdown",
+      "metadata": {},
       "source": [
+        "<style>\n",
+        "    #codecell4 {\n",
+        "        y-overflow: scroll;\n",
+        "        max-height: 200px;\n",
+        "    }\n",
+        "</style>\n",
+        "\n",
         "[View the runnable example on GitHub](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/inference/tensorflow/accelerate_tensorflow_inference_onnx.ipynb)"
-      ],
-      "metadata": {}
+      ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {},
       "source": [
         "# Accelerate TensorFlow Inference using ONNXRuntime"
-      ],
-      "metadata": {}
+      ]
     },
     {
       "cell_type": "markdown",
+      "metadata": {},
       "source": [
         "You can use ``InferenceOptimizer.trace(..., accelerator='onnxruntime')`` API to enable the ONNXRuntime acceleration for TensorFlow inference. It only takes a few lines."
-      ],
-      "metadata": {}
+      ]
     },
     {
       "cell_type": "markdown",
-      "source": [
-        "To apply ONNXRuntime acceleration, the following dependencies need to be installed first:"
-      ],
       "metadata": {
         "nbsphinx": "hidden"
-      }
+      },
+      "source": [
+        "To apply ONNXRuntime acceleration, the following dependencies need to be installed first:"
+      ]
     },
     {
       "cell_type": "code",
-      "source": [
-        "# for BigDL-Nano\n",
-        "!pip install --pre --upgrade bigdl-nano[tensorflow,inference]\n",
-        "!source bigdl-nano-init\n",
-        "\n",
-        "# for ONNXRuntime acceleration\n",
-        "!pip install onnx onnxruntime "
-      ],
-      "outputs": [],
       "execution_count": null,
       "metadata": {
         "nbsphinx": "hidden"
-      }
+      },
+      "outputs": [],
+      "source": [
+        "!pip install --pre --upgrade bigdl-nano[tensorflow,inference] # install the nightly-built version\n",
+        "!source bigdl-nano-init # set environment variables"
```

Comment:
we should remind users to `source bigdl-nano-init` before starting jupyter kernel, it's too late to source here

Reply:
Yes, we have a reminder in the next markdown cell to reminds user to `source bigdl-nano-init` before the kernel is started, which will be visible when user open this notebook locally, from online preview tools, etc. :)